subprojects/logging/src/test/groovy/org/gradle/internal/logging/console/DefaultScalableElapsedTimeFormatterTest.groovy

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.logging.console
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class DefaultScalableElapsedTimeFormatterTest extends Specification {
+    def formatter = new DefaultScalableElapsedTimeFormatter()
+
+    @Unroll
+    def "shows #output when elapsed time is greater or equals than #lowerBoundInclusive but lower than #upperBoundExlusive"() {
+        when:
+        def result = formatter.format(input);
+
+        then:
+        result == output
+
+        where:
+        lowerBoundInclusive | upperBoundExclusive | input            | output
```

Comment:
Good point, let's discuss the passing the `Clock` instance and see if we still need this.